src/coreclr/src/pal/src/CMakeLists.txt

```diff
@@ -68,37 +68,37 @@ if(CLR_CROSS_COMPONENTS_BUILD)
     if(CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL x86_64 OR CMAKE_HOST_SYSTEM_PROCESSOR STREQUAL amd64)
         if(CLR_CMAKE_TARGET_ARCH STREQUAL "arm" OR CLR_CMAKE_TARGET_ARCH STREQUAL "armel")
             if($ENV{CROSSCOMPILE} STREQUAL "1")
-                set(PAL_CMAKE_PLATFORM_ARCH_I386 1)
+                set(CLR_CMAKE_HOST_ARCH_I386 1)
```

Comment:
Is this whole block really needed here ?

If I am reading the cmake files correctly, we should be getting the definitions from configurecompiler.cmake. `add_subdirectory(src/pal)` is after `include(configurecompiler.cmake)` in src\coreclr\CMakeLists.txt.